File: python/python-list-of-objects/example4.py
class Student:
    def __init__(self, name, age):
        self.name = name
        self.age = age


l = []
l.insert(0, Student("Alex", 20))
l.insert(1, Student("Bob", 21))
l.insert(2, Student("Ira", 15))

for item in l:
    print("Name : {}, Age : {}".format(item.name, item.age))

File: javascript/javascript-iterate-array-of-objects/example4.js
// https://codevscolor.com/javascript-iterate-array-of-objects

const arr = [{id: 1, name: 'Alex', age: 20}, {id: 2, name: 'Bob', age: 22}, {id: 3, name: 'Chandler', age: 21}];

arr.forEach((item, index)=> {
    console.log(`Item at index ${index}: `, item);
});